src/pickers/DatePicker.js

```diff
@@ -44,11 +60,11 @@ class AdDatePicker extends Component {
       readOnly = false,
       required = false,
       fullWidth = true,
-      format = "YYYY-MM-DD",
+      format = "DD-MM-YYYY",
       reset,
       ...otherProps
     } = this.props;
-
+    if (this.props.isSecondaryCalendarEnabled){
```

Comment:
Destructure _isSecondaryCalendarEnabled_ from props and then use it in if statement:
```
    const {
      intl,
      classes,
      disablePast,
      module,
      label,
      readOnly = false,
      required = false,
      fullWidth = true,
      format = "DD-MM-YYYY",
      reset,
      **isSecondaryCalendarEnabled**,
      ...otherProps
    } = this.props;

if (**isSecondaryCalendarEnabled**) {...}
```